locales/ja.json

```diff
@@ -45,6 +49,10 @@
       "description": "すべてのARIA progressbarノードにアクセシブルな名前があることを確認します",
       "help": "ARIA progressbarノードにはアクセシブルな名前がなければなりません"
     },
+    "aria-prohibited-attr": {
+      "description": "Ensures ARIA attributes are not prohibited for an element's role",
+      "help": "Elements must only use permitted ARIA attributes"
+    },
```

Comment:
@ma10 Did you mean to leave these without a translation?

Reply:
No, it should be something that has been added to the base branch after my last update and I overlooked  when I merged the base branch last time.
I'll try to work on that one as soon as I get a chance.

Reply:
Done with commit 1c2d6695